openstates/cli/update.py

```diff
@@ -502,6 +502,9 @@ def parse_args() -> tuple[argparse.Namespace, list[str]]:
     # realtime mode
     parser.add_argument("--realtime", action="store_true", help="enable realtime mode")
 
+    # kafka mode
+    parser.add_argument("--kafka", type=str, help="Enable Kafka (MSK) integration")
```

Comment:
Instead of "Enable Kafka (MSK) integration" I think something closer to "enable writes to Kafka" would make more sense